boards/arm/nrf5340dk_nrf5340/nrf5340dk_nrf5340_cpunet.yaml

```diff
@@ -10,3 +10,4 @@ ram: 64
 flash: 256
 supported:
   - watchdog
+  - netif
```

Comment:
Afaik, in contrary to the name, we don't do any networking on the net core, it just runs the driver.

Reply:
Should this capability remove for this board?